tasks/main.yml

```
@@ -80,3 +80,6 @@
     mode: '0644'
   notify:
     - "Restart miniflux"
+
+- name: Import backup-restore
+  ansible.builtin.import_tasks: backup-restore.yml
```

Personally I manage the backups in a different role so I would make it configurable, so enable this task only if a flag is turned on.
```suggestion
- name: Import backup-restore
  ansible.builtin.import_tasks: backup-restore.yml
  when: miniflux_backup_enabled
```

---

done